ChemistryLib/PhreeqcIOData/AqueousSolution.h

```diff
@@ -24,7 +24,7 @@ namespace PhreeqcIOData
 {
 struct Component
 {
-    explicit Component(std::string name_) : name(std::move(name_)) {}
+    explicit Component(std::string&& name_) : name(std::move(name_)) {}
```

Comment:
minor: can be kept as value which is more general. *i.e* on can pass something constant to the ctor.

Reply:
kept the ctor as it was.